scripts/src/owners/owners_file.py

```diff
@@ -8,70 +9,67 @@
     from yaml import SafeLoader
 
 
+class OwnersFileError(Exception):
+    pass
+
+
 def get_owner_data(category, organization, chart):
     path = os.path.join("charts", category, organization, chart, "OWNERS")
-    status, owner_content = get_owner_data_from_file(path)
-    return status, owner_content
+    success = True
+
+    try:
+        owner_content = get_owner_data_from_file(path)
+    except OwnersFileError as e:
+        print(f"Error getting OWNERS file data: {e}")
+        success = False
+
+    return success, owner_content
```

Comment:
Nit: Can we assert success instead of asserting failure?
```suggestion
    success = False

    try:
        owner_content = get_owner_data_from_file(path)
        success = True
    except OwnersFileError as e:
        print(f"Error getting OWNERS file data: {e}")

    return success, owner_content
```